vendor.mod

```diff
@@ -177,7 +177,7 @@ replace (
 	github.com/matttproud/golang_protobuf_extensions => github.com/matttproud/golang_protobuf_extensions v1.0.1
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.6.0
 	github.com/prometheus/procfs => github.com/prometheus/procfs v0.0.11
-	github.com/vishvananda/netlink => github.com/vishvananda/netlink v1.1.0
+	github.com/vishvananda/netlink => github.com/vishvananda/netlink v1.2.1-beta.2
```

Comment:
We can remove this replace rule if we decide to update to a new release; ISTR we added this to not update to a non-released version if there was no need. As there's a (beta) release now, perhaps we should consider updating to a newer version either way 🤔